### DISCUSSION TITLE
Multiple select

### DISCUSSION THREAD

**Discussion by gumiranda** [tamagui/tamagui#179] [Ideas]
I need a multiple select to implement a list of checkboxes that can be selected by the user .

The ui kitten library have this solution, i think that could be useful in tamagui too.

**Comment by mmtftr** [comment]
I've implemented a multiple select component below with minimal modifications. It is with Tamagui v1.94.5

https://github.com/mmtftr/jellip/blob/2f337e6393a6c6281444e8ab0e215b410791386d/components/SelectBox.tsx

Those that are interested can try to copy/paste and it should work (I've tested on Native).

I think the focus features might be broken on web though. Plus, I'm pretty sure some of the APIs that I've used are subject to change in the future.
